pages/index.js

```diff
@@ -204,6 +202,21 @@ export default function Home() {
           height: 1em;
         }
 
+        .button-primary {
+            color: (33, 33, 33);
```

Comment:
this should be: `color: rgb(33, 33, 33);` (right now the `rgb` is missing)

or to keep consistent with the other color values, `color: rgba(33, 33, 33, 1);`

Reply:
Whoops - easy fix. Should be good now.